src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -149,23 +158,34 @@ public void run(String[] args) throws Exception {
 			addConfluentProperties(props);
 		}
 
+		KAFKA_AUTO_COMMIT_INTERVAL_MS = getEnvironmentVariable("KAFKA_AUTO_COMMIT_INTERVAL_MS", "1000");
+		KAFKA_ENABLE_AUTO_COMMIT = getEnvironmentVariable("KAFKA_ENABLE_AUTO_COMMIT", "false");
+		KAFKA_SESSION_TIMEOUT_MS = getEnvironmentVariable("KAFKA_SESSION_TIMEOUT_MS", "30000");
+
 		props.put("group.id", group);
-		props.put("enable.auto.commit", "false");
-		props.put("auto.commit.interval.ms", "1000");
-		props.put("session.timeout.ms", "30000");
+		props.put("enable.auto.commit", KAFKA_ENABLE_AUTO_COMMIT);
+		props.put("auto.commit.interval.ms", KAFKA_AUTO_COMMIT_INTERVAL_MS);
+		props.put("session.timeout.ms", KAFKA_SESSION_TIMEOUT_MS);
 
 
 		boolean depositToS3 = false;
 		AmazonS3 s3 = null;
 		AmazonKinesisFirehoseAsync firehose = null;
-		if (destination != null && destination.equals("s3")) {
+		Storage gcsStorage = null;
+
+		if (destination.equals("s3")) {
 			depositToS3 = true;
 			s3 = createS3Client(awsRegion);
-
-		} else {
+		} else if (destination.equals("firehose")) {
 			firehose = buildFirehoseClient(awsRegion);
+		} else if (destination.equals("gcs")) {
+			gcsStorage = StorageOptions.getDefaultInstance().getService();
```

Comment:
Maybe add a comment here to indicate it uses the `GOOGLE_APPLICATION_CREDENTIALS ` environment variable by default?

Reply:
I've added a comment here!